app/stacks/applications-service/variables/dev.hcl

```diff
@@ -0,0 +1,3 @@
+locals {
+  mysql_database = "ipclive"
```

Comment:
Should this be `ipcdev`?

Reply:
That's the current value in the existing dev!

Reply:
Will be finding out the real values soon. So for now these are ok, especially seeing as we have no way to connect to the actual database!